playwright/support/helper.ts

```diff
@@ -193,6 +193,17 @@ export const containsClass = async (
   await expect(locatorFunc).toHaveClass(classNameRegEx);
 };
 
+/**
+ * Uses the .toHaveFocus() assertion with a waitFor before to
+ * ensure the locator is available, and enough time has passed for the focus to be set.
+ * @param locatorFunc the locator you'd like to use
+ * @example await toBeFocusDelayed(exampleLocator(page));
+ */
+export const toBeFocusedDelayed = async (locatorFunc: Locator) => {
+  await locatorFunc.waitFor({ timeout: 10000 });
```

Comment:
**question**: Purely for my own knowledge, why 10 seconds? Why not e.g. 5? Just wondering whether it came from your investigation or whether it's just a number plucked from the aether.

Reply:
I believe 5 is the standard, so double that just to account for extra flakiness. Depending on what the second reviewer thinks, I'm happy to let this be higher or lower though 👍